.github/workflows/build-test-ship.yaml

```diff
@@ -21,14 +21,16 @@ on:
         required: true
       SLACK_BOT_TOKEN:
         required: true
+      AWS_ACCOUNT_ID:
+        required: true
 
 jobs:
   # Retrieve the minimum Go version from go.mod and output a matrix of all Go
   # versions released since. Skip on release.
   go-versions:
     name: Get supported Go versions
     runs-on: ubuntu-latest
-    if: ${{ !github.event.release }}
+    if: ${{ github.event_name != 'release' }}
```

Comment:
You explained the reasoning on this before, but I'm forgetting what it was 🤔 mind sharing again?

Reply:
This is IMO the more proper way to check what type of event it is. GitHub guarantees it will give the event name in the context for the job, so it is more proper to check an explicit value in a guaranteed field than it is to check truthiness of an object that may or may not exist in the context. Functionally this code is the same for releases, but it works more consistently across all event types we might use.